scripts/generators.js

```diff
@@ -1,10 +1,16 @@
 var utils = require('../lib/utils');
 
-hexo.extend.generator.register('blog-post-introducing-aframe', function () {
-  return {
-    path: 'blog/2015/12/16/0.0.10-release/',
-    data: utils.createRedirectResponse(hexo, 'blog/2015/12/16/introducing-aframe/')
-  };
+hexo.extend.generator.register('blog-redirects', function () {
+  return [
+    {
+      path: 'blog/2015/12/16/0.0.10-release/',
+      data: utils.createRedirectResponse(hexo, 'blog/introducing-aframe/')
+    },
```

Comment:
can you add another redirect for `blog/2015/12/16/introducing-aframe/` -> `blog/introducing-aframe/`?